### DISCUSSION TITLE
Factura com IVA 14% e 5%, como fica no TaxPayable?

### DISCUSSION THREAD

**Discussion by Helkyd** [assoft-portugal/SAF-T-AO#179] [Q&A] [answered]
Boa tarde a todos,
Tenho uma Factura com 3 itens dos quais o primeiro tem 14% e os outros dois tem 5% de IVA.

    <Line>
     <LineNumber>1</LineNumber>
     <ProductCode>0212</ProductCode>
     <ProductDescription>Frango fresco</ProductDescription>
     <Quantity>12.72</Quantity>
     <UnitOfMeasure>Unidade</UnitOfMeasure>
     <UnitPrice>2206.15</UnitPrice>
     <TaxPointDate>2021-04-30</TaxPointDate>
     <Description>0212</Description>
     <CreditAmount>28062.23</CreditAmount>
     <Tax>
      <TaxType>IVA</TaxType>
      <TaxCountryRegion>AO</TaxCountryRegion>
      <TaxCode>NOR</TaxCode>
      <TaxPercentage>14</TaxPercentage>
     </Tax>
    </Line>
    <Line>
     <LineNumber>2</LineNumber>
     <ProductCode>02126</ProductCode>
     <ProductDescription>Fuba de Bombó kg</ProductDescription>
     <Quantity>1.20</Quantity>
     <UnitOfMeasure>Unidade</UnitOfMeasure>
     <UnitPrice>762.00</UnitPrice>
     <TaxPointDate>2021-04-30</TaxPointDate>
     <Description>02126</Description>
     <CreditAmount>914.40</CreditAmount>
     <Tax>
      <TaxType>IVA</TaxType>
      <TaxCountryRegion>AO</TaxCountryRegion>
      <TaxCode>NOR</TaxCode>
      <TaxPercentage>5</TaxPercentage>
     </Tax>
    </Line>
    <Line>
     <LineNumber>3</LineNumber>
     <ProductCode>0212616</ProductCode>
     <ProductDescription>Fuba de Bombó</ProductDescription>
     <Quantity>1.00</Quantity>
     <UnitOfMeasure>Unidade</UnitOfMeasure>
     <UnitPrice>762.00</UnitPrice>
     <TaxPointDate>2021-04-30</TaxPointDate>
     <Description>0212616</Description>
     <CreditAmount>762.00</CreditAmount>
     <Tax>
      <TaxType>IVA</TaxType>
      <TaxCountryRegion>AO</TaxCountryRegion>
      <TaxCode>NOR</TaxCode>
      <TaxPercentage>5</TaxPercentage>
     </Tax>
    </Line>
    <DocumentTotals>
     **<TaxPayable>2890.04</TaxPayable>**
     <NetTotal>29738.63</NetTotal>
     <GrossTotal>32628.67</GrossTotal>
    </DocumentTotals>

Ao validar na AGT do Produtor:
**DocumentTotals.TaxPayable (FR 21/1) está mal calculado.**

Qual o nosso erro?

Obrigado pelo suporte.

**Comment by cryptolopes** [comment, marked as answer]
Talvez o `TaxCode`, uma vez que a taxa de 5% está indicada como sendo `NOR`.
Para além disso o `TaxPayable` está efetivamente mal calculado. ([link para comentário](https://t.me/saftao/5964))
